cmd/kube-proxy/app/server.go

```diff
@@ -451,7 +452,7 @@ with the apiserver API to configure the proxy.`,
 		klog.Fatalf("unable to create flag defaults: %v", err)
 	}
 
-	opts.AddFlags(cmd.Flags())
+	opts.AddFlags(legacyflag.InitFlagSet(cmd.Flags()))
```

Comment:
I'd recommend just changing the underlying flagset for `cmd.Flags()` to legacyflag as well. If we do that, I'm not sure we need the new `InitFlagSet` method (see my comment here https://github.com/kubernetes-sigs/legacyflag/pull/3#issuecomment-510516128).

For example: If `cmd.Flags().Parse(os.Args)` is how these are getting parsed, and we later extend legacyflag's Parse method to do something special, this will need to be updated again to use the legacyflag flagset at the top level rather than the plfag flagset (e.g. make cmd.Flags()'s underlying flagset a `legacyflag.FlagSet`). Starting out with a full conversion to legacyflag would help us avoid these kinds of inconsistencies down the road.